README.md

```diff
@@ -60,6 +60,7 @@ module "account_setup" {
 | enable_rule_require_cloud_trail | Specifies if 'Cloud Trail enabled' rule should be enabled | string | `false` | no |
 | enable_rule_require_root_account_MFA | Specifies if 'Require root account MFA enabled' rule should be enabled | string | `false` | no |
 | enable_rule_require_tag | Specifies if 'Require Tag' rule should be enabled | string | `false` | no |
+| event_selector | Specifies an event selector for enabling data event logging, It needs to be a list of map values. See: https://www.terraform.io/docs/providers/aws/r/cloudtrail.html for details on this map variable | list | `<list>` | no |
```

Comment:
Did you generate the docs using terraform docs, looks like the default not matching the empty list as in the vars.